src/Components/Blazor/Blazor/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -72,6 +79,45 @@ private WebAssemblyHostBuilder()
         /// </summary>
         public IServiceCollection Services { get; }
 
+        /// <summary>
+        /// Registers a <see cref="IServiceProviderFactory{TBuilder}" /> instance to be used to create the <see cref="IServiceProvider" />.
+        /// </summary>
+        /// <param name="factory">The <see cref="IServiceProviderFactory{TBuilder}" />.</param>
+        /// <typeparam name="TBuilder">The type of builder provided by the <see cref="IServiceProviderFactory{TBuilder}" />.</typeparam>
+        public void ConfigureContainer<TBuilder>(IServiceProviderFactory<TBuilder> factory)
```

Comment:
Is it named `ConfigureContainer` as a reference to the term "DI container"? That seems like a non-sequitur, since our terminology is "service provider". How about `ConfigureServiceProviderFactory`?

Reply:
Oh I see this is meant to be equivalent to the server-side `ConfigureContainer` method.

I'm not always in favour of aiming for API symmetry between server and WebAssembly since the scenarios and capabilities are so different, but on this occasion it seems reasonable. It's a sufficiently obscure API that having it be a bit eccentric is not a big deal.

Reply:
yeah I didn't want to make this totally different.